src/main/com/algos/src/maths/AbsoluteMax.java

```diff
@@ -13,12 +13,12 @@ public static int getMaxValue(int... numbers) {
         if (numbers == null || numbers.length == 0) {
             throw new IllegalArgumentException("Numbers array cannot be empty or null");
         }
-        int absMax = numbers[0];
+        int absMax = 0;  
         for (int i = 1; i < numbers.length; i++) {
-            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
+            if (numbers[i] > absMax) {
```

Comment:
**Issue**: The condition 'numbers[i] > absMax' fails to find the maximum absolute value as it does not consider negative numbers properly. It only compares the raw values, disregarding whether a negative value might have a higher absolute value than the current 'absMax'. <br> **Fix**: Modify the condition to compare the absolute values of 'numbers[i]' and 'absMax'. This ensures that the method correctly identifies the maximum absolute value. <br> **Code Suggestion**: 
 ```
 -            if (numbers[i] > absMax) {
 +            if (Math.abs(numbers[i]) > Math.abs(absMax)) {
 ```